code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -170,20 +171,32 @@
 	hud_possible = list (DIAG_STAT_HUD, DIAG_BATT_HUD, DIAG_MECH_HUD, DIAG_TRACK_HUD)
 
 /obj/item/radio/mech //this has to go somewhere
+	subspace_transmission = TRUE
```

Comment:
?

Reply:
Please clarify what you mean here

Reply:
Why set subspace_transmission to TRUE? am I missing something?

Reply:
It was manually set previously when the radio was added